src/mono/netcore/System.Private.CoreLib/src/System/Globalization/GlobalizationMode.Mono.cs

```diff
@@ -2,14 +2,13 @@
 // The .NET Foundation licenses this file to you under the MIT license.
 // See the LICENSE file in the project root for more information.
 
+using System.Diagnostics.CodeAnalysis;
+
 namespace System.Globalization
 {
     internal static partial class GlobalizationMode
     {
-        private static bool GetInvariantSwitchValue() =>
-            GetSwitchValue("DOTNET_SYSTEM_GLOBALIZATION_INVARIANT");
-
```

Comment:
The linker will need a bool method which it can transform to false to disable ICU support at app build time.

Reply:
Does the existing `GlobalizationMode.get_Invariant` property work for this?

Reply:
It does.

Reply:
FWIW, we're still doing something similar: https://github.com/dotnet/runtime/blob/b0ed2bdf8e163dc8c38277c8fa27a82df4b820e9/src/libraries/System.Private.CoreLib/src/System/Globalization/GlobalizationMode.Unix.cs#L17-L32

Reply:
@vargaz does this look good, or is there any action I should take?

Reply:
I merged this, but if something else need to happen here, I'm happy to put up a follow up PR.